pkg/registry/softwarecomposition/networkneighbors/etcd.go

```diff
@@ -0,0 +1,41 @@
+package networkneighbors
+
+import (
+	"github.com/kubescape/storage/pkg/apis/softwarecomposition"
+	"github.com/kubescape/storage/pkg/registry"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/registry/generic"
+	genericregistry "k8s.io/apiserver/pkg/registry/generic/registry"
+	"k8s.io/apiserver/pkg/registry/rest"
+	"k8s.io/apiserver/pkg/storage"
+)
+
+// NewREST returns a RESTStorage object that will work against API services.
+func NewREST(scheme *runtime.Scheme, storageImpl storage.Interface, optsGetter generic.RESTOptionsGetter) (*registry.REST, error) {
+	strategy := NewStrategy(scheme)
+
+	dryRunnableStorage := genericregistry.DryRunnableStorage{Codec: nil, Storage: storageImpl}
+
+	store := &genericregistry.Store{
+		NewFunc:                   func() runtime.Object { return &softwarecomposition.NetworkNeighbors{} },
+		NewListFunc:               func() runtime.Object { return &softwarecomposition.NetworkNeighborsList{} },
+		PredicateFunc:             MatachNetworkNeighbor,
+		DefaultQualifiedResource:  softwarecomposition.Resource("networkneighborses"),
+		SingularQualifiedResource: softwarecomposition.Resource("networkneighbors"),
```

Comment:
thought: can we use only plural for resources instead of `networkneighborses`?

That sure sounds ugly and is not proper grammar. What if we just provide the `DefaultQualifiedResource` to be `networkneighbors` and refer to it like that?

Reply:
I don't get the suggestions. How should a single resource be called?